docs/DeveloperGuide.adoc

```diff
@@ -725,13 +725,74 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |add a new person |
 
+|`* * *` |user |add description to my contacts |I can make my contacts feel unique
+
 |`* * *` |user |delete a person |remove entries that I no longer need
 
+|`* * *` |user |find a person by email |locate details of persons without having to go through the entire list
+
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |clear the storage in the address book |restart and empty the address book
+
+|`* * *` |user |find a person by phone number |locate details of persons without having to go through the entire list
+
+|`* * *` |user |find a person by tag |locate details of persons without having to go through the entire list
+
+|`* * *` |user |list my contacts by their tags |locate my contact more easily
+
+|`* * *` |user |find a person by date added |locate details of a persons without having to go through the entire list
+
+|`* *` |user |rename tag function |rename a specified tag from all contacts with that tag
+
+|`* *` |user |remove tag function |remove a specified tag from all contacts with that tag
+
+|`* *` |user |have an edit contact function |do not have to delete the contact and recreate contact when changing a person's particulars
+
+|`* *` |user |add multiple phone numbers to one contact |do not have to create another contact to add a person's secondary number(s)
+
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
+|`* *` |user |to create favourite lists of contacts |find my most important contacts easily
```

Comment:
can remove the `to`. How does this user stories differ from having a tag called `favourites`? :)